spa/src/components/common/RedirectWithReload.js

```diff
@@ -0,0 +1,9 @@
+import { useHistory, withRouter } from 'react-router-dom';
+
+const RedirectWithReload = (props) => {
+  let history = useHistory();
```

Comment:
`withRouter` already passes the `history` to the component.
```suggestion
const RedirectWithReload = ({ history, to, location}) => {
```

Reply:
Addressed with 02f12cff0cdaa189fe4b9b93678a7b00068cf129